portfolio/src/main/java/com/google/sps/data/VideoAnalysis.java

```diff
@@ -16,6 +20,7 @@ public VideoAnalysis(Video video) throws IOException {
         SearchQuery searchQuery = new SearchQuery(video, SEARCH_QUERY_SIZE);
         this.sentimentScore = sentimentAnalysis.getScore();
         this.sentimentMagnitude = sentimentAnalysis.getMagnitude();
```

Comment:
I would recommend leaving it as a LocalDate object.  Looks like the Period function accepts two local dates as a function call.

https://docs.oracle.com/javase/8/docs/api/java/time/Period.html#between-java.time.LocalDate-java.time.LocalDate-